12/umbraco-commerce/reference/storefront-api/endpoints/README.md

```diff
@@ -0,0 +1,43 @@
+# Storefront API Endpoints

```

Comment:
I'm wondering whether this landing page is necessary?
Wouldn't it be a little cleaner if the endpoints where direct subpages to the Storefront API README article?

Reply:
I was thinking it could be handy to just have a page to be able to point to for all the endpoints without having to scroll down a long main page. Also, linked to the SUMMARY file I wondered if it kept that less cluttered, but you can use your judgment on that 👍

Reply:
Ah! That's a good point. Let's keep the landing page there.

But all the articles need to be added to the SUMMARY file though.
Otherwise, they will not be part of the site on GitBook I'm afraid 😅

Reply:
Do you want me to add them? Or you OK adding them in the same order as defined in the endpoints page?

Reply:
I'm on it 💪

Reply:
Legend 🙌